README.md

```diff
@@ -28,6 +28,9 @@ Or extend the existing functionality to suit your needs (e.g. Download files fro
     SETUP:
     /install - Install chromedriver & Selenium webdriver. Run this once.
 
+    GLOBAL: (accepted with every command)
+    /custom_browser [appdata_local_routing] - Use custom browser, Input should be the routing inside %appdatalocal% dir (e.g. "Microsoft\Edge")
```

Comment:
The convention is just a singleword, so replace custom_browser with `browserdir` or `custombrowser` or `appdir`

Also, I'd change text to "Use custom Chromium-based browser by providing its directory name within %appdatalocal%"

Reply:
Done